hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -398,22 +398,22 @@ export class MissingContainers extends React.Component<Record<string, object>, I
           <Tabs defaultActiveKey='1'>
             <Tabs.TabPane
               key='1'
-              tab={`Missing${(missingDataSource && missingDataSource.length > 0) ? ` (${missingDataSource.length})` : ''}`}>
+              tab={`Missing${(missingDataSource?.length > 0) ? ` (${missingDataSource.length})` : ''}`}>
```

Comment:
I think we can remove the check and shorten it to:
```suggestion
              tab={`Missing ${missingDataSource?.length ?? ''}`}>
```

Reply:
@devabhishekpal 
Done chnages in latest commit 878b46ef6f3090e43a104f606bdf49ad890f0e4f